index.html

```diff
@@ -1,82 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Kashif Saleem Portfolio Web Page</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Kashif Saleem Portfolio Web Page</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
 </head>
+
 <body>
   <!----------- The Top Nav bar section for desktop app -------------------- -->
   <header>
-    <nav class="container">
+    <!-- <nav class="container">
       <div id="bars">
         <button><i class="fa fa-bars" style="color:black;"></i></button>
       </div>
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and more readable 😉